test/export/StructureDefinition.ResourceExporter.test.ts

```diff
@@ -269,12 +270,21 @@ describe('ResourceExporter', () => {
         /FHIR prohibits logical models and resources from constraining parent elements. Skipping.*at path 'language'.*File: ConstrainParent\.fsh.*Line:\D*/s
       );
     });
+  });
 
-    expect(exported.name).toBe('MyTestResource');
-    expect(exported.id).toBe('MyResource');
-    expect(exported.type).toBe('MyResource');
-    expect(exported.baseDefinition).toBe('http://hl7.org/fhir/StructureDefinition/DomainResource');
-    expect(exported.elements).toHaveLength(12); // 9 AlternateIdentification elements + 3 added elements
```

Comment:
Is there a reason these 5 expect statements were removed? If there wasn't, can they be added back in? I think we want them to ensure the resource is being exported correctly.

Reply:
@guhanthuran can you add these lines back in? They should still be included because your PR does not change the test above the one you added. They should be added back in after line 270.

Reply:
These lines are still not added back in. This test should still be testing those expect statements. The warning above this line in the diff view of the PR shows that the `exported` value isn't used because these expect statements aren't included in the test. I'm happy to help today if you want.